Assets/Scripts/Object/Spawnable/Astronaut.cs

```diff
@@ -1,3 +1,38 @@
-﻿namespace Object.Spawnable {
-    public class Astronaut : AbstractObject { }
+﻿using Object.Manager;
+using UnityEngine;
+using UnityEngine.Serialization;
+using Utils;
+
+
+namespace Object.Spawnable {
+    public class Astronaut : AbstractObject { 
+
+        [SerializeField] private AudioSource hitSound1;
+        [SerializeField] private AudioSource hitSound2;
+        
+        private bool hitsound1played;
+    
+        private AudioSource audioSource; 
+        void Start(){
+
+            audioSource = GetComponent<AudioSource>();
```

Comment:
![ask](https://img.shields.io/badge/review-ask-9b71ae.svg)
Are you using `audioSource`? If not, I'd like you to delete this code and definition on the above

Reply:
Oh! Good point, I have removed it now!